packages/manager/modules/pci/src/projects/project/kubernetes/details/service/service.html

```diff
@@ -1,18 +1,44 @@
 <cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+<oui-message data-type="info" data-dismissable="false" data-ng-if="!$ctrl.cluster.isUpToDate">
+    <span data-translate="kube_service_cluster_update_available"></span>.
+    <a href data-ng-disabled="$ctrl.cluster.status !== $ctrl.STATUS.READY" data-ng-bind="::'kube_service_common_update' | translate" data-ng-click="$ctrl.updateCluster()"></a>
```

Comment:
Should be a `button` with `oui-link` class

Reply:
Using a button causes the UI to go for a toss, mainly because of the padding that the button hsa

Reply:
This has been replaced by a button now :)